### DISCUSSION TITLE
Preferred recording method

### DISCUSSION THREAD

**Discussion by pan-zhuo** [uwlib-cams/sinopia_maps#90] [seeking feedback]
Is it a good practice to always set up an IRI input type for a property whenever RDA allows for it, i.e., use a Lookup/URI template for any property that can take IRIs as values?

It seems that there's an order of usefulness of recording methods: IRI, identifier, structured description, unstructured description. See https://github.com/uwlib-cams/MARC2RDA/wiki/2022-Meeting-Minutes#008-mapping-review

Problem is that catalogers may wish to record unstructured descriptions to help users identify and find a resource, and this seems to be the current PCC practice for properties like Place of publication. (See [MGD: Place of publication](https://www.loc.gov/aba/rda/mgd/manifestation/mg-m-placeOfPublication-01.pdf))

However, I believe the [Manifestation statements](https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-6f241594-3b46-3eb3-a053-1ecec038fa34) serve the purpose of identifying and finding. (See discussion #89)

Granted, there is the complication of cataloger-supplied data, and in this case I think you can only use a literal for something like Place of publication since it's not on the manifestation itself.

But catalogers can always choose to input a literal in Sinopia even if the template asks for an IRI. We will just need to provide clear guidance in remarks. (How to make the remarks more visible to make sure the cataloger notices them is a whole other issue...)

**Comment by briesenberg07** [comment]
> Is it a good practice to always set up an IRI input type for a property whenever RDA allows for it, i.e., use a Lookup/URI template for any property that can take IRIs as values?

Seems that this is what we want to do, but that given limitations on external data sources/lookup sources can be used in well-formed RDA, there will be times when a lookup property template isn't feasible.
